packages/rn-tester/Podfile.lock

```diff
@@ -1404,66 +1409,66 @@ EXTERNAL SOURCES:
-  ScreenshotManager: d9c4ee0cfd08bf51f3d8eb0e53d6dc25846331f2
+  React-Core: fd2885c2d067aa8dc9631607fdd2126eb88f0989
+  React-CoreModules: 6e13f394b7e30cf36ee834eb636f110fe890b6fe
+  React-cxxreact: c77e643263440789acf2fc9607b8978c348665a9
+  React-debug: f24314b94a0c6277a04af68cb5d7d80770283ec9
+  React-Fabric: 0f97d60642726966ec9edc6a91aae97d7ef122ad
+  React-FabricImage: 0e80649617ca22c3828f08819d1d4a9464dbe3bb
+  React-graphics: 81d54d2a035ed527fdab45fab2866de466500b28
+  React-hermes: 9253c3659dcae4497c588bb15dbd9a8fe310400f
+  React-ImageManager: 77febd35292dee0a70a456f84abd17036a01a14e
+  React-jserrorhandler: c88d5eba1a0fa1668a7ee6c7923d9362f49492f9
+  React-jsi: 7c325a70bc23fa1f2f137e4d34852f322aa6079c
+  React-jsiexecutor: 1c01bd1a5cc65d57dddb45b1a686a4c58b63a134
+  React-jsinspector: d995804706bc81b4be3514537700cbec3cc85df9
+  React-jsitracing: 07c56984bc087698b85323f8d0b0c03781300101
+  React-logger: 3c744900fd728feb7de1018ff30dd1492614e54f
+  React-Mapbuffer: c27ff793d0e1844516936e820dc1892bd265d55c
+  React-nativeconfig: 8bd6a5b3a1621f9b32b12b0a3133ac07cfb5d9c1
+  React-NativeModulesApple: e3aa17a9104f2f2dee1af01b5554fe2843abc75c
+  React-perflogger: 28aa8a42b4f12387bc0bcc755972b491c59a98e5
+  React-RCTActionSheet: a328f97881bde5bd5516a8831b2f01c128575b36
+  React-RCTAnimation: 22ed945e444117f630313651e04a75039a645564
+  React-RCTAppDelegate: 56ee8810b24befc42faa86da5b35d6e14555e3c2
+  React-RCTBlob: 6e7cabaa64f58eea4fba56e489daa548f323df4f
+  React-RCTFabric: c967d9125957bdaf4c7669d69e35ddad98e5fefa
+  React-RCTImage: 40ad7dbc86d6443eb4303fd312c69e6466e95a79
+  React-RCTLinking: 80d6f05117e42b16ddc97dd817ad0eae40168976
+  React-RCTNetwork: bee3e473c808305e37e68358703103b318ee5cc4
+  React-RCTPushNotification: 43da38f8fe4daddeb3b375e6350dd299af8ff0b9
+  React-RCTSettings: f9d1e65d0415ab363a3fa4dfb32c2aaef7e9893b
+  React-RCTTest: 1e24c6b7b600b8fb8d183add109b1e1f77ca4fbd
+  React-RCTText: 020916a1b5414a43c5c96fa2b7c7b4499280cd40
+  React-RCTVibration: 96076539d91fb6b2b23f50ea0e1c7b02a19c3c60
+  React-rendererdebug: cc531dc5612a0c884554e65de0dfbf2da939f432
+  React-rncore: a510360c74981cf5302a4cf2aecaf7103e6ffdfa
+  React-RuntimeApple: d85735cdd2312b7bdd0da1dc8e8e388af8eb221c
+  React-RuntimeCore: 7f9153841d67055e8035c6c779a884304834798c
+  React-runtimeexecutor: c296ce7c8f7c5c446bc2289bbbf9171e4230d742
+  React-RuntimeHermes: f915554131b595e7bad85174b207d4232099d227
+  React-runtimescheduler: 9d38e9c33a398a5d98dcc9bdb01110966393bbe6
+  React-utils: a5d32c7836fc2cf583ba1a46acc3d96136c57992
+  ReactCommon: ad2eded2b983eb106bebdc7ad1e3bffa0497cf2c
+  ReactCommon-Samples: 38d4438605dd5aa75fbc5988bda99295f4de8694
+  ScreenshotManager: a9eeb34fe4aa282612135566eaf13bfa61be65fd
   SocketRocket: abac6f5de4d4d62d24e11868d7a2f427e0ef940d
-  Yoga: ba060e7ec094e57aa646d6413e40608303556227
+  Yoga: 333040b09e1ed6d5ac8dc80864edf9b4fb44ef3b
 
 PODFILE CHECKSUM: 5afcf37691103b83159fb73be088b7cadc67af7b
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.14.3
```

Comment:
Did you accidentally upgraded CocoaPods version to use?

cc @cipolleschi - should we stick with 1.13.0?

Reply:
Yep, we should revert the Podfile.lock, @Saadnajmi

Reply:
Sure. Related, I can make a separate PR to update?

Reply:
Nope, because AFAIR, Cocoapods 1.14 only works with Ruby 3, and we still need to support Ruby 2

Reply:
Fixed, but also, looks like 1.14.3 _does_ support Ruby 2: https://github.com/CocoaPods/CocoaPods/releases/tag/1.14.3

Reply:
uh, thanks for reporting! I think we can doing it for RNTester, but I won't do it for the template as this will force a non required migration to the whole ecosystem. WDYT?

Reply:
Makes sense, I can do that later. I don't think 1.14 has anything I need (visionOS is in 1.13 I believe).